data/xml/Q17.xml

```diff
@@ -406,71 +465,90 @@
-    <href>https://transacl.org/ojs/index.php/tacl/article/download/1170/267</href>
     <pages>441–454</pages>
-    <url>http://www.aclweb.org/anthology/Q17-1031</url>
+    <url>https://www.aclweb.org/anthology/Q17-1031</url>
   </paper>
 
   <paper id="1032">
-    <title>Unsupervised Acquisition of Comprehensive Multiword Lexicons using Competition in an n-gram Lattice</title>
+    <title>Unsupervised Acquisition of Comprehensive Multiword Lexicons using Competition in an </title>
     <author><first>Julian</first><last>Brooke</last></author>
-    <author><first>Jan</first><last>Snajder</last></author>
+    <author><first>Jan</first><last>Šnajder</last></author>
     <author><first>Timothy</first><last>Baldwin</last></author>
     <year>2017</year>
+    <month>December</month>
+    <doi>10.1162/tacl_a_00073</doi>
     <abstract>We present a new model for acquiring comprehensive multiword lexicons from large corpora based on competition among n-gram candidates. In contrast to the standard approach of simple ranking by association measure, in our model n-grams are arranged in a lattice structure based on subsumption and overlap relationships, with nodes inhibiting other nodes in their vicinity when they are selected as a lexical item. We show how the configuration of such a lattice can be optimized tractably, and demonstrate using annotations of sampled n-grams that our method consistently outperforms alternatives by at least 0.05 F-score across several corpora and languages.</abstract>
-    <href>https://transacl.org/ojs/index.php/tacl/article/download/1186/268</href>
-    <pages>455–47032</pages>
-    <url>http://www.aclweb.org/anthology/Q17-10</url>
-    <video href="https://vimeo.com/277673914" tag="video"/>
+    <pages>455–470</pages>
+    <url>https://www.aclweb.org/anthology/Q17-1032</url>
+    <video href="https://vimeo.com/277673914" tag="video" />
   </paper>
 
   <paper id="1033">
-    <title>Replicability Analysis for Natural Language Processing: Testing Significance with Multiple Datasets</title>
+    <title>Replicability Analysis for Natural Language Processing: Testing
+                    Significance with Multiple Datasets</title>
```

Comment:
Funky newline in title.

Reply:
...maybe I'd afford eating something other than Soylent. This is idiopathic to a few records in 2017.